### ISSUE TITLE
Recurring Error When Training with Google Colab

### ISSUE THREAD

**Issue by nmarafo** [rhasspy/piper#234]
Description:
When trying to train a model using the piper project in a Google Colab notebook, I encounter several errors and warnings. Below, I detail the process I followed and the error I encountered.

Code used:
```
from google.colab import drive
drive.mount('/content/drive')

!git clone https://github.com/rhasspy/piper.git
%cd piper/src/python

!pip install -r /content/piper/src/python/requirements.txt
!bash build_monotonic_align.sh
!pip install -e .

!sed -i "s/self.session = onnxruntime.InferenceSession(onnx_path)/self.session = onnxruntime.InferenceSession(onnx_path, providers=['AzureExecutionProvider', 'CPUExecutionProvider'])/g" /content/piper/src/python/piper_train/norm_audio/vad.py

!python3 -m piper_train.preprocess \
  --language en-us \
  --input-dir /input_path/ \
  --output-dir /output_path/ \
  --dataset-format ljspeech \
  --single-speaker \
  --sample-rate 22050

!pip install torchmetrics==0.11.4
!pip install torchtext==0.14.0

!python3 -m piper_train \
    --dataset-dir /my_dataset_path/ \
    --accelerator 'gpu' \
    --devices 1 \
    --batch-size 32 \
    --validation-split 0.0 \
    --num-test-examples 0 \
    --max_epochs 10000 \
    --resume_from_checkpoint /my_ckpt_path \
    --checkpoint-epochs 1 \
    --precision 32 
```
Error encountered:
```
/usr/local/lib/python3.10/dist-packages/torchvision/io/image.py:13: UserWarning: Failed to load image Python extension: ...
... (other warning and debug messages) ...
corrupted size vs. prev_size
```
I'd like to know if anyone else has experienced this error and if there is any solution or recommendation regarding this. Thanks for your help.

**Comment by rmcpantoja** [comment]
Hi @nmarafo, 
You can use the updated notebooks and fixes for VAD in #177 (my fork). Microsoft updated onnxruntime a time ago, and the update Habe breaking changes that has been solved in this PR.

**Comment by nmarafo** [comment]
"It works perfectly. Great job.
Thank you very much!!"
